### DISCUSSION TITLE
Cloudreve Android Application.

### DISCUSSION THREAD

**Discussion by Saisnat** [cloudreve/cloudreve#2117] [Ideas]
Exactly the title.
Android application when?

**Comment by Polv89** [comment]
I follow

**Comment by nanzhiwhy** [comment]
https://yun.urldwz.com/s/qOmzFL

**Reply by cabaseira** [reply]
Hi!
Thanks for the information.
What is that app? Is it official Cloudreve support?

**Comment by Lato-o** [comment]
any news ?
